create_env.py

```diff
@@ -38,6 +38,7 @@
     "FIRSTRUN_AUTO_ADMIN_PASSWORD": "",
     "ELKS46_API_USER": "",
     "ELKS46_API_KEY": "",
+    "MAKERSPACE_LOCAL_TIMEZONE": "Europe/Stockholm",
```

Comment:
This should be a prompt with a good default. It should link to where one can find all valid timezones, and it should validate that the timezone the user sets is a valid timezone.